core/services/vrf/v2/integration_v2_test.go

```diff
@@ -189,7 +189,7 @@ func newVRFCoordinatorV2Universe(t *testing.T, key ethkey.KeyV2, numConsumers in
 	backend := cltest.NewSimulatedBackend(t, genesisData, gasLimit)
 	h, err := backend.Client().HeaderByNumber(testutils.Context(t), nil)
 	require.NoError(t, err)
-	require.LessOrEqual(t, math.MaxInt64, h.Time)
```

Comment:
the type changed here to a uint64 and require can't handle that without overflowing. Just removed the check as I can't see what value it was providing